src/ceph-volume/ceph_volume/api/lvm.py

```diff
@@ -137,6 +137,9 @@ def get_lv_from_argument(argument):
     in the form of `vg/lv`, but with some validation so that an argument that is a full
     path to a device can be ignored
     """
+    if argument.startswith('/'):
+        lv = get_lv(lv_path=argument)
+        return lv
```

Comment:
This condition would also match a raw device or partition like ``/dev/sdc1``. Should we check to see if an lv is found here before returning it?

Reply:
It is safe to return this directly because `/dev/sdc1` would be a `None`, the checks are usually done in consumers of this helpers since behavior might be different, although it is usually in the form of:

    if get_lv_from_argument(args.data):
        do_lv_related_operations
    else:
        check_if_arg_is_a_device

Reply:
Ok, returning ``None`` here should be just fine.